### PR TITLE
Build displayname from givenName and surname to suit our displayName builder.

### PR DESCRIPTION
**Pull request by henninghaakonsen**

### CODE REVIEW

**Comment by tronghn** [comment]
Isn't displayName already in the same format?

**Comment by henninghaakonsen** [comment]
@tronghn In our db it is, but from azure it is surname, givenName.

**Comment by tronghn** [comment]
Huh. I'm seeing the correct format when testing locally against Azure 🤷‍♂

File: src/main/kotlin/no/nav/dsop/kontroll/service/azure/Routes.kt [hunk 1]
```diff
@@ -26,6 +26,10 @@ fun Route.azure(graphService: GraphService) {
                 searchParam,
                 true
             )
+        }.map {
+            val reassignedMember = it
+            reassignedMember.displayName = "${it.givenName} ${it.surname}"
+            reassignedMember
```
Comment (tronghn) [inline src/main/kotlin/no/nav/dsop/kontroll/service/azure/Routes.kt]: We can avoid making the `displayName` field mutable (that is, `var` vs `val`) by rewriting to this:

```kotlin
.map { reassignedMember ->
    reassignedMember.copy(displayName = "${reassignedMember.surname} ${reassignedMember.surname}")
}
```

**Pull request merged**